src/operations/sepia_tone.rs

```diff
@@ -0,0 +1,18 @@
+use crate::{arg_parsers::SepiaThreshold, error::MagickError, image::Image};
+
+pub fn sepia_tone(image: &mut Image, _threshold: &SepiaThreshold) -> Result<(), MagickError> {
+    let mut rbga = image.pixels.to_rgba32f();
+    for pixel in rbga.pixels_mut() {
+        let r = pixel[0];
+        let g = pixel[1];
+        let b = pixel[2];
+
+        // German wikipedia shows coefficients for digital sepia toning.
```

Comment:
We'll need to study the imagemagick implementation more to match it better.

Sepia is really just desaturate + tint, so what we need out of imagemagick is the color it uses, and its handling of the percentages - I wouldn't put it past it to do something weird, like fully desaturate and not tint at all at intensity 0%, or who knows what else.